web/src/components/tasks/anytime-action/tax-clearance-certificate/helpers.ts

```diff
@@ -0,0 +1,73 @@
+import { LookupMunicipalityByName } from "@businessnjgovnavigator/shared/municipality";
```

Comment:
Created this file just to move some logic out of the parent components that was making it difficult to track what was happening.